### ISSUE TITLE
-v doesn't report as an invalid arg

### ISSUE THREAD

**Issue by cfjedimaster** [sintaxi/surge#104]
I always forget the argument to get Surge's version. I'll do this:

    surge -v

Which for some reason acts as if I'm doing a new site. If I do 

    surge -x

I get a proper result: 

    surge: `--x` is not a surge command.

**Comment by silentrob** [comment]
IMO, `-v` should give the version which Ive always felt was more natural than `-V`

**Comment by JimFung** [comment]
I see that verbose is an available option in the code, I think the intention was to have -V and -v map to "version" and "verbose", respectively, since I'm not sure I made -v invalid. 
Personally I like -v for version too, I always forget to capitalize.

**Comment by silentrob** [comment]
Yes, I think `-v` was borrowed from ssh which support `-v` and adds extra verbosity with `-vv` and `-vvv` I'm not sure if there are any stats on how often verbose is used, but making that change would be hard as it breaks backwards compat, even if it is low risk.
